remote/remote_test.go

```diff
@@ -1470,10 +1470,15 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 				h.AssertNil(t, origImage.SetLabel("some-label", "some-val"))
 				h.AssertNil(t, origImage.Save())
 
+				identifier, err := origImage.Identifier()
+				h.AssertNil(t, err)
+
+				repoID := identifier.String()
+
 				img, err = remote.NewImage(
-					repoName,
+					repoID,
 					authn.DefaultKeychain,
-					remote.FromBaseImage(repoName),
+					remote.FromBaseImage(repoID),
```

Comment:
Without this change, this test was failing saying that after deletion, the image was still present. My assumption is that `ggcr/registry` treats deletions differently that `docker-distribution`. 

I think `remote.Image.Delete()` deletes by the identifier but `remote.Image.Found()` checks by tag and `docker-distribution` cleans up "hanging" tags whereas `ggcr/registry` does not.

That's just my assumption, I'd like to hear other ideas or suggestions.